ml-agents-envs/mlagents/envs/environment.py

```diff
@@ -495,8 +498,6 @@ def step(
                 else:
                     if text_action[brain_name] is None:
                         text_action[brain_name] = [""] * n_agent
-                    if isinstance(text_action[brain_name], str):
```

Comment:
Note this is an actual functional change.  Requires that if we are given a `text_action` that it will be a `Dict[str, List[str]]`.  Otherwise the types will be inconsistent.  I don't believe we use this feature but I thought it was worth noting.